app/views.py

```diff
@@ -34,10 +34,6 @@
 
 from . import app, db
 
-from .config import (ADMINS, DEFAULT_MAIL_SENDER, BASEDIR, SEED_FILES, 
```

Comment:
These constants should be referenced from the current running app instance, not imported directly from a `config.py` file—see example below.